src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -23,9 +23,11 @@ public abstract partial class ViewHandler : IViewHandler
 			[nameof(IView.Height)] = MapHeight,
 			[nameof(IView.IsEnabled)] = MapIsEnabled,
 			[nameof(IView.Semantics)] = MapSemantics,
-			Actions = {
-					[nameof(IFrameworkElement.InvalidateMeasure)] = MapInvalidateMeasure
-				}
+			Actions =
+			{
+				[nameof(IFrameworkElement.InvalidateMeasure)] = MapInvalidateMeasure,
+				[nameof(ViewHandler.NeedsContainer)] = MapNeedsContainer,
```

Comment:
This is an action that can be fired from within the handler to update the `HasContainer`.